lib/index.js

```diff
@@ -20,19 +23,25 @@ module.exports = function (arg1, arg2) {
 
       generateOptions.sourceMap = createSourceMap
 
-      rollup(rollupOptions).then((bundle) => {
-        const result = bundle.generate(generateOptions)
-
-        file.contents = Buffer.from(result.code)
-
-        if (createSourceMap) {
-          result.map.file = entry
-          result.map.sources = result.map.sources.map(source => path.relative(file.cwd, source))
-          applySourceMap(file, result.map)
-        }
-
-        cb(null, file)
-      })
+      rollup(rollupOptions)
+        .then(bundle => {
+          const result = bundle.generate(generateOptions)
+
+          file.contents = Buffer.from(result.code)
+
+          if (createSourceMap) {
+            result.map.file = entry
+            result.map.sources = result.map.sources.map(source => path.relative(file.cwd, source))
+            applySourceMap(file, result.map)
+          }
+
+          cb(null, file)
+        }).catch(err => {
+          process.nextTick(() => {
+            this.emit('error', new PluginError(PLUGIN_NAME, err))
+            cb(null, file)
```

Comment:
In the context of Node.js, the error should be passed to the 1st argument of callback if it exists. Also I wonder that we should pass the `file` to the 2nd arugment of the callback since the processing is failed when it enters in this block.

I think we can simply write like below:

```js
cb(new PluginError(PLUGIN_NAME, err))
```